### DISCUSSION TITLE
How we can add pin drop using Maplibre-gl

### DISCUSSION THREAD

**Discussion by Vida-Subhash** [maplibre/maplibre-gl-js#1327] [Q&A]
Hello everyone. 
Thank you one of the best Map library.

My question is that how i can added pin drop to the map.

**Comment by HarelM** [comment]
Not sure exactly what you mean, but I would recommend checking the examples in the docs. If it's something standard you'll find it there.

**Reply by Vida-Subhash** [reply]
Actually I want to drop pin on Map, how can i achieve this.

**Reply by wipfli** [reply]
https://maplibre.org/maplibre-gl-js-docs/example/add-a-marker/
